### PR TITLE
Add slash command support and added ltp command as our first

### PR DESCRIPTION
**Pull request by CodyFraker**

### CODE REVIEW

File: Bloon/Core/Discord/Bot.cs [hunk 1]
```diff
@@ -71,6 +80,8 @@ public override async Task Enable()
             this.cNext.RegisterCommands<GeneralCommands>();
             this.cNext.RegisterCommands<OwnerCommands>();
 
+            this.slash.RegisterCommands<LTPSlashCommand>(Guilds.SBG);
```
Comment (dukeofsussex) [inline Bloon/Core/Discord/Bot.cs:83]: Move to feature

File: Bloon/Features/LTP/LTPSlashCommand.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        Timestamp = DateTime.UtcNow,
+                    });
+
+                    await this.db.SaveChangesAsync();
+                    await ctx.CreateResponseAsync(InteractionResponseType.ChannelMessageWithSource, new DiscordInteractionResponseBuilder().WithContent("You have been added to the Looking to Play role!"));
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```
Comment (dukeofsussex) [inline Bloon/Features/LTP/LTPSlashCommand.cs:58]: We have a logger for this

File: Bloon/Features/LTP/LTPSlashCommand.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            this.db = db;
+        }
+
+        [SlashCommand("ltp", "Add or remove yourself from the Looking To Play role.")]
+        public async Task LTPAsyncslash(InteractionContext ctx)
+        {
+            DiscordRole ltp = ctx.Guild.GetRole(SBGRoles.LookingToPlay);
+            DiscordMember guildUser = await ctx.Guild.GetMemberAsync(ctx.User.Id);
+            try
```
Comment (dukeofsussex) [inline Bloon/Features/LTP/LTPSlashCommand.cs:28]: Wrapping the entire code in a try-catch are we? Pretty sure we already have a listener handling command errors.

**Review by dukeofsussex — CHANGES_REQUESTED** [review]

File: Bloon/Core/Discord/Bot.cs (hunk 1 above)
Comment (CodyFraker) [reply Bloon/Core/Discord/Bot.cs:83]: can't doesn't work for whatever purpose.

File: Bloon/Features/LTP/LTPSlashCommand.cs (hunk 3 above)
Comment (CodyFraker) [reply Bloon/Features/LTP/LTPSlashCommand.cs:28]: whatever dude

File: Bloon/Features/LTP/LTPSlashCommand.cs (hunk 2 above)
Comment (CodyFraker) [reply Bloon/Features/LTP/LTPSlashCommand.cs:58]: eypic 8)

**Review by dukeofsussex — APPROVED** [review]

**Pull request merged**